courses/free-courses-en.md

```diff
@@ -400,6 +401,12 @@
 * [IBM Cloud Essentials V3](https://cognitiveclass.ai/courses/ibm-cloud-essentials) - CognitiveClass.ai
 
 
+### Openstack
+
+* [Openstack CERN Admin guide](https://clouddocs.web.cern.ch/index.html) - CERN
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. I would just put this directly under cload computin and not create a separate category

Reply:
np, moved the Openstack guide to books in the cloud computing category.